src/cryptonote_core/blockchain.cpp

```diff
@@ -975,6 +975,118 @@ bool Blockchain::init(
         }
     }
 
+    {
```

Comment:
Could we move this chunk of code into db_sqlite.cpp so that there's just one place to look for all possible queries?  I do realize the intention is eventually to remove this, but I'm thinking it might be a little while before we can (because people could have dormant service nodes that aren't active and don't get upgraded until months from now and reintroduce the problem if this code doesn't stick around for a while).